impeller/typographer/text_run.cc

```diff
@@ -17,6 +17,7 @@ TextRun::~TextRun() = default;
 
 bool TextRun::AddGlyph(Glyph glyph, Point position) {
   glyphs_.emplace_back(GlyphPosition{glyph, position});
+  has_color_ |= glyph.type == Glyph::Type::kBitmap;
```

Comment:
Its not that they're always bitmaps though. The COLR fonts can be drawn as a path too

Reply:
Should I rename this to something like `is_bitmap` or something then?

Reply:
the underlying is_color is being used to detect both bitmaps and COLR fonts right now though

Reply:
there was a skia parameter that returned whether or not something was a bitmap but I couldn't figure out how it was initialized, and seemed to always be false even for emoji